IFC4x3/Templates/Project Context/Project Global Positioning Geodetic/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Project_Global_Positioning_Geodetic_3rJ3aKTBj11wk7gKUHSOa" Name="Project Global Positioning Geodetic" UniqueId="f54c3914-74bb-4107-ab87-fea51e45c624" Type="IfcContext">
+	<Rules>
+		<DocModelRuleAttribute Name="RepresentationContexts">
+			<Rules>
+				<DocModelRuleEntity Name="IfcGeometricRepresentationContext">
+					<Rules>
+						<DocModelRuleAttribute Name="HasCoordinateOperation" Identification="HasGlobalPosition">
+							<Rules>
+								<DocModelRuleEntity Name="IfcRigidOperation">
+									<Rules>
+										<DocModelRuleAttribute Name="TargetCRS">
+											<Rules>
+												<DocModelRuleEntity Name="IfcGeographicCRS">
+													<Rules>
+														<DocModelRuleAttribute Name="HasWellKnowTextRepresentation" Identification="WKTRepresentation">
+															<Rules>
+																<DocModelRuleEntity Name="IfcWellKnownText">
+																	<Rules>
```

Comment:
Same comment as in the other PR. WKT should be a partial but then we merge without the partial first and create the partial on top.